.tekton/build-service-pull-request.yaml

```diff
@@ -234,6 +299,47 @@ spec:
         operator: in
         values:
         - "true"
+    - name: build-sealights-container
```

Comment:
Conversely, in the PR pipeline, maybe we can build _only_ the sealights container?

Reply:
currently this is a super good idea! Thank you!